app/collins/models/RemoteAssetFinder.scala

```diff
@@ -48,15 +48,15 @@ case class AssetSearchParameters(
   def toSeq: Seq[(String, String)] = {
     val q1: Seq[(String, String)] = (
       params._1.map { case (enum, value) => (enum.toString, value) } ++
-      params._2.map { case (assetMeta, value) => ("attribute" -> "%s;%s".format(assetMeta.name, URLEncoder.encode(value, "UTF-8"))) } ++
-      params._3.map { i => ("attribute" -> ("ip_address;" + URLEncoder.encode(i, "UTF-8"))) }) ++ afinder.toSeq :+ ("details" -> (if (details) "true" else "false"))
+      params._2.map { case (assetMeta, value) => ("attribute" -> "%s;%s".format(assetMeta.name, value)) } ++
```

Comment:
Can you explain what is going on here? Is it safe to not URL encode the strings we're sending across over HTTP?